### ISSUE TITLE
Request to add example layouts

### ISSUE THREAD

**Issue by UnsolvedCypher** [chakra-ui/chakra-ui#1281]
It's a little difficult for a new user to get started with Chakra UI without seeing a few non-trivial example pages. Semantic UI has some [example layouts](https://react.semantic-ui.com/layouts) with accompanying code that make it easy to see how to build a nice-looking page using Semantic UI's components.

I think Chakra UI would really benefit from a similar page with just a few starter layouts to help people get started.

**Comment by kstulgys** [comment]
Hey, I would love to create some sample layouts. Can I get more details/instructions on how to submit a PR to this? Where does the code go?

**Comment by kstulgys** [comment]
Hey @with-heart can you provide more guidelines?

**Comment by UnsolvedCypher** [comment]
Hey @kstulgys thanks for offering to take this on! I'm not @with-heart but I think a good place to start may be to create a `layouts` folder in the `examples` folder of this repo. Once a PR with some example layouts is merged, then perhaps it can be added to the website.

**Comment by with-heart** [comment]
@kstulgys Sorry I hadn't gotten back to you. Haven't been around much the last month, too much going on at work. I'm back now but haven't had a chance to get caught up on my github notifications yet!

@UnsolvedCypher That's an excellent plan. I'd definitely love to make them available on the site, but placing them in the codebase or creating some codesandbox examples and collecting them here would be a great start till we figure out what to do with them!

**Comment by umarluqman** [comment]
I would like to add the form layout from my portfolio website, https://umarluqman.dev/showcase/complex-form as one of the layouts, let me know what you guys think.

Currently, it's on v0.8, will be using the v1 for MR later

**Comment by waptik** [comment]
> I would like to add the form layout from my portfolio website, [umarluqman.dev/showcase/complex-form](https://umarluqman.dev/showcase/complex-form) as one of the layouts, let me know what you guys think.
> 
> Currently, it's on v0.8, will be using the v1 for MR later

I think you should create codesandbox versions of your layouts and put the links here.
@with-heart Wouldn't it be simplier to rename this issue or create a new issue for examples and pin it for easy access ?
It'll be a temporary thing until it some of them make it to the docs/guides

**Comment by aditya-mitra** [comment]
Guys, what happened to this issue?

**Comment by ivanlhz** [comment]
Hello there, I am working on example layouts. Please, consider check it out and add to the doc/guides. On the other hand, I will make more examples in a few days. Thank you in advance.
- [Login Page](https://codesandbox.io/s/chakra-ui-login-page-ncc3q)
- [Basic product list](https://codesandbox.io/s/chakra-ui-product-list-1b7pq)

**Comment by segunadebayo** [comment]
Thanks for requesting this @UnsolvedCypher. After much thought, we won't be adding this to the docs in the near term, even though I admit that it's a nice-to-have.

I'd recommend checking out Chakra UI Pro (http://pro.chakra-ui.com/) to see what's possible when it comes to layout. Also check the codesandbox ideas shared by @ivanlhz (Thank you, man!)

Cheers.
